src/app/js/fields/FieldCompletes.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field } from 'redux-form';
+import MenuItem from 'material-ui/MenuItem';
+import translate from 'redux-polyglot/translate';
+
+import FormSelectField from '../lib/components/FormSelectField';
+import {
+    polyglot as polyglotPropTypes,
+    field as fieldPropTypes,
+} from '../propTypes';
+
+const styles = {
+    complete: {
+        paddingLeft: '40px',
+    },
+    header: {
+        lineHeight: '48px',
+        paddingLeft: 0,
+    },
+};
+
+const FieldComplete = ({ fields, p: polyglot }) => {
+    const otherFieldsMenuItems = fields.map(f => (
```

Comment:
what's the interest of extracting it from the JSX if you don't reuse it or test it?